src/sentry/api/endpoints/organization_projects.py

```diff
@@ -102,17 +102,26 @@ def get(self, request, organization):
 
         queryset = queryset.filter(status=ProjectStatus.VISIBLE).distinct()
 
-        return self.paginate(
-            request=request,
-            queryset=queryset,
-            order_by=order_by,
-            on_results=lambda x: serialize(
-                x,
-                request.user,
-                ProjectSummarySerializer(
-                    environment_id=self._get_environment_id_from_request(request, organization.id),
-                    stats_period=stats_period,
+        # TODO(davidenwang): remove this after frontend requires only paginated projects
+        get_all_projects = request.GET.get("all_projects") == "1"
+
+        if get_all_projects:
+            sorted_projects = sorted(list(queryset), key=lambda x: x.slug)
+            return Response(serialize(sorted_projects, request.user, ProjectSummarySerializer()))
+        else:
+            return self.paginate(
+                request=request,
+                queryset=queryset,
+                order_by=order_by,
+                on_results=lambda x: serialize(
+                    x,
+                    request.user,
+                    ProjectSummarySerializer(
+                        environment_id=self._get_environment_id_from_request(
+                            request, organization.id
+                        ),
+                        stats_period=stats_period,
```

Comment:
Do you need stats_period or environment_id in the non paginated case?

Reply:
Nope I don't believe so as I just need the same summary as was returned in org details https://github.com/getsentry/sentry/blob/b0760de2666a7378a3e6e4fd8d08b847083b3767/src/sentry/api/serializers/models/organization.py#L254